src/main/scala/circt/stage/Annotations.scala

```diff
@@ -101,6 +101,8 @@ case class EmittedMLIR(
 
 }
 
+case class FirtoolBinaryPath(option: String) extends NoTargetAnnotation with CIRCTOption
```

Comment:
This should probably have a ScalaDoc as well as a shell option form

Reply:
I added a scaladoc... couldn't quite figure out how to add a shell option but if you are invoking from a shell you can always do `PATH=path/to/firtool:$PATH <command>` so not sure if it is 100% necessary.